src/app/modules/lists/dual-listbox/dual-listbox.component.html

```diff
@@ -0,0 +1,122 @@
+<div class="dual-list">
+	<div class="listbox">
+    @if (sourceName(); as sourceName) {
+      <p>{{ sourceName }}</p>
+    }
+
+		<div class="record-picker">
```

Comment:
The formatting is off.

Reply:
That's weird, in VS code i see this:

<img width="908" alt="Screenshot 2024-10-30 at 11 44 03" src="https://github.com/user-attachments/assets/de74df28-2de3-42cb-a75a-78696b1c1e8a">